src/Form/RedirectForm.php

```diff
@@ -76,7 +76,7 @@ public function form(array $form, FormStateInterface $form_state) {
         '#type' => 'language_select',
         '#title' => t('Language'),
         '#languages' => Language::STATE_ALL,
```

Comment:
we should actually be able to drop this completey, it should get a widget that should just work..?